config/clock.rb

```diff
@@ -53,6 +53,7 @@ class Clock
   every(1.day, 'Generate export for TAD', at: '23:59') { DataAPI::TADExport.run_daily }
 
   every(1.day, 'SendEocDeadlineReminderEmailToCandidatesWorker', at: '12:00') { SendEocDeadlineReminderEmailToCandidatesWorker.new.perform }
+  every(1.day, 'SendNewCycleStartedEmailToCandidatesWorker', at: '12:00') { SendNewCycleHasStartedEmailToCandidatesWorker.new.perform }
```

Comment:
I've followed the same pattern here as the `SendEocDeadlineReminderEmailToCandidatesWorker` - are we happy to have this cron job firing indefinitely (even though the email will only be sent on one day of the year)?

Reply:
This seems sub-optimal but the only other thing i can think of is manually triggering seems worse. Wdyt?